packages/related-keyphrase-suggestions/src/components/KeyphrasesTable/index.js

```diff
@@ -198,9 +198,11 @@ export const KeyphrasesTable = ( { columnNames = [], data, renderButton, related
 					</div>
 				</Table.Header>
 
-				{ renderButton && <Table.Header className="yst-w-32">
-					<div className="yst-flex yst-justify-end yst-text-right rtl:yst-text-left">
-						{ __( "Add keyphrase", "wordpress-seo" ) }
+				{ renderButton && <Table.Header>
+					<div className="yst-flex yst-justify-end yst-w-full">
```

Comment:
I don't think this `yst-w-full` does anything?

Reply:
Dropped it